showcase/app/templates/components/dropdown.hbs

```diff
@@ -1368,4 +1368,23 @@
       {{! template-lint-enable no-inline-styles }}
     </SF.Item>
   </Shw::Flex>
+</section>
+
+<Shw::Divider />
+
+<section>
+  <Shw::Text::H2>Demo</Shw::Text::H2>
+
+  <Shw::Grid {{style gap="2rem"}} @columns={{2}} as |SG|>
+    <SG.Item @label="With fixed width">
```

Comment:
should we have a fixed height as well?

Reply:
Yes, probably. I'll add more tests while I do my explorations, to see what other use cases are missing, and potentially open a new PR with those.